samples/boards/ti/cc13x2_cc26x2/system_off/src/ext_flash.c

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (c) 2020, Texas Instruments Incorporated
+ * Copyright (c) 2020 Linaro Ltd.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <device.h>
+#include <drivers/gpio.h>
+
+#include <driverlib/cpu.h>
+
+#define GPIO_PORT DT_TI_CC13XX_CC26XX_GPIO_40022000_LABEL
+#define DIO8_PIN  8
```

Comment:
Could these be named to reflect their use, rather than their value?  E.g. `SPI_CLK_PIN` and `FLASH_CS_PIN` instead of `DIO10_PIN` and `DIO20_PIN`?

Reply:
Thanks for the review! I think I'd like to keep the style, comments and naming in this file so that it is reasonably similar to the original code I borrowed from the TI SimpleLink SDK. This would help ease future comparison and maintenance.

Reply:
Comparison yes, maintenance no.  IMO.